src/module/storage/NetworkStorage.ts

```diff
@@ -124,7 +124,17 @@ export const NetworkStorage = {
         const networks = NetworkStorage.getStorage();
         const masterUuid = Helpers.uuidForStorage(master.uuid);
         const slaveUuids = networks[masterUuid] ?? [];
-        return slaveUuids.map(uuid => fromUuidSync(Helpers.uuidFromStorage(uuid))) as (SR5Actor | SR5Item)[];
+        const slaves = [] as (SR5Actor | SR5Item)[];
```

Comment:
it would be more sound to use it as `const slaves: (SR5Actor | SR5Item)[] = [];`